worlds/tloz/__init__.py

```diff
@@ -165,9 +166,10 @@ def create_items(self):
                 if item_name == starting_weapon:  # To remove the extra Sword if that's our weapon
                     item_amounts[item_name] -= 1
                 if self.multiworld.TriforceLocations[self.player] > 0 or item_name != "Triforce Fragment":
-                    i = 0
                     for i in range(0, item_amounts[item_name]):
                         self.multiworld.itempool.append(self.create_item(item_name))
+                    if item_name == "Bomb":
+                        self.multiworld.itempool[-1].classification = ItemClassification.progression
```

Comment:
correct me if im wrong, but wouldn't this make EVERY bomb progression? i know we need to mark at least one bomb as progression for logic, so we should probably mark the one that gets placed in a shop up at the beginning of this function as such then. that might be the cause of the "cant place all advancement items" issue

Reply:
[-1] accesses the last item in a list. This is done after all the bombs are added, and only modifies the last one added

Reply:
ok gotcha